doc/source/whatsnew/v1.5.0.rst

```diff
@@ -432,6 +432,7 @@ Other API changes
   <https://developers.googleblog.com/2022/02/making-oauth-flows-safer.html?m=1#disallowed-oob>`_.
   The ``auth_local_webserver = False`` option is planned to stop working in
   October 2022. (:issue:`46312`)
+- :func:`read_json` now raises ``FileNotFoundError`` (previously ``ValueError``) when input is a string ending in ``.json``, ``.json.gz``, ``.json.bz2``, etc. but no such file exists. (:issue:`29102`)
```

Comment:
move to the I/O section

Reply:
I think this is technically an API change since the exception class changed.

Reply:
It was under whatsnew **IO** originally but moved to **API changes** following https://github.com/pandas-dev/pandas/pull/46718#discussion_r846662440.